src/server/room.js

```diff
@@ -44,11 +45,27 @@ class Room {
 
   constructor(id /* : string */) {
     this.id = id;
+    this.playerList = [];
+    this.teamNone = { ...defaultTeam};
     this.teamA = { ...defaultTeam };
     this.teamB = { ...defaultTeam };
     this.phase = 'team-a-coding';
     this.round = 1;
   }
+
+  addPlayerToRoom(userId /*: string */) {
+    if (this.playerList.includes(userId)) {
+      console.log(`ROOM ${this.id}: failed to add player ${userId}, already in playerList`)
+      return false;
+    }
+    this.playerList.push(userId)
```

Comment:
push a `new Player(userId)` (not sure if Player is implemented)

Reply:
i shall have to implement Player, i will add a TODO for now